app/src/main/java/io/apicurio/registry/ccompat/store/RegistryStorageFacadeImpl.java

```diff
@@ -192,6 +201,13 @@ public CompatibilityCheckResponse testCompatibilityBySubjectName(String subject,
         });
     }
 
+    /**
+     * Given a content removes any quoted brackets. This is useful for some validation corner cases in avro where some libraries detects quoted brackets as valid and others as invalid
+     */
+    private ContentHandle removeQuotedBrackets(String content) {
+        return ContentHandle.create(content.replace("\"{}\"", "{}"));
```

Comment:
Maybe we should add the semicolon as well, to make sure the replaced value is not a key? e.g. `: *\"\{}\"`
The downside is the code would have to use regex instead of a simple string replace.

Reply:
Done